src/libs/ReportUtils.js

```diff
@@ -727,7 +737,7 @@ function getIcons(report, personalDetails, defaultIcon = null, isPayer = false)
         result.source = Expensicons.DeletedRoomAvatar;
         return [result];
     }
-    if (isThread(report)) {
+    if (isChatThread(report)) {
```

Comment:
Maybe but this is just a method name refactor. The functionality is the same as before so I'm not updating the logic here since its outside of the scope of this PR which already has complex test steps.